remixer_core/src/main/java/com/google/android/libraries/remixer/Remixer.java

```diff
@@ -117,7 +120,7 @@ public void addItem(RemixerItem remixerItem) {
     }
     listForKey.add(remixerItem);
     remixerItem.setRemixer(this);
-    remixerItems.add(remixerItem);
+    getRemixerItemsForContext(remixerItem.getContext()).add(remixerItem);
```

Comment:
This method depends on the precondition that `remixerItem` has a context. Can you add a check at the top of the method to assert that this is the case?